Vagrantfile

```diff
@@ -1,42 +1,80 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
 
-HOSTNAME = 'ansible-st2'
-
 VAGRANTFILE_API_VERSION = '2'
+VIRTUAL_MACHINES = {
+  :u14 => {
```

Comment:
`u14` is assumption we made internally, but for other users of this repo and contributors it could be confusing.
I think `vagrant up ubuntu14` is more expected here.